java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -220,6 +232,11 @@ private static LoadFlowParametersPointer convertToLoadFlowParametersPointer(Load
             Network network = ObjectHandles.getGlobal().get(networkHandle);
             LoadFlowParameters parameters = createLoadFlowParameters(dc, loadFlowParametersPtr);
             String providerStr = CTypeUtil.toString(provider);
+            if (providerStr.equals("")) {
+                providerStr = PyPowsyblConfiguration.getLoadFlowProvider();
+            }
+            Logger rootLogger = (Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
```

Comment:
We should not log it in the root logger but in the class logger (arg to be replaced with PyPowsyblApiLib.class), same for other logs.